src/index.js

```diff
@@ -137,9 +137,9 @@ module.exports = class Bitwap {
       cb(err, block)
     }
 
-    this.getBlocks([key], (err, res) => {
-      if (err) {
-        return done(err)
+    this.getBlocks([key], (errs, res) => {
+      if (errs) {
+        return done(errs[0])
       }
 
       done(null, res[0])
```

Comment:
the method is called `getBlock` so yes we only want a single block, and will only receive back a single block.